content/blog/2020-04-27-moving-to-the-matrix/index.md

```diff
@@ -0,0 +1,163 @@
+  and the French government, to name a few.
+- Matrix instances can connect to each other ("federation"). This means from our
+  GNU Radio chat, we can access other organizations on Matrix, even if they're
+  not part of the GNU Radio chat instance. This includes the main Matrix instance
+  on [matrix.org](https://matrix.org), where projects such as our friends from
+  SatNOGS are located.
+- That means that people already on Matrix don't have to do a whole lot to join
+  our Matrix server.
+- Bridging to IRC is also easy, and will have a high likelihood of working
+  indefinitely.
+- The open standards nature of Matrix, and the availability of open source
+  reference clients, means that there are lots of clients out there, including
+  things like a [Weechat plugin](https://github.com/poljar/weechat-matrix).
+
+
+## The matter of Transparency, or why am I only hearing about this now?
+
+While the GNU Radio leadership team definitely did their due diligence in
+picking a new service, one thing that definitely shouldn't become the norm for
+decisions around the GNU Radio project is the fact that we decided this behind
+closed doors within the leadership team. For example, we did not publish a GREP
+ahead of time for a public discussion.
+
+In this case, we were under a bit of time pressure, and the choice of chat service
+seems to be something where a lot of personal preference comes in. To be blunt,
+if we were going to allow everyone to argue on behalf of their favourite messenger,
+we wouldn't have been able to resolve this on time.
+
+## What happens to the GNU Radio Slack?
+
+The Slack plan we're on right now is free, and there's no reason to turn it off.
+However, all important discussions regarding the project will be held on our
+Matrix instance going forward, in order to preserve them in history.
+We are paying for a one-time unlocking of the the Slack history, and will make
+them available in some form to all participants of the GNU Radio chat.
+
+Many people from the GNU Radio leadership will be leaving the Slack server, and
+won't be reachable there going forward.
+
+## The new setup, and yes, we're still tuning it
+
+As of April 2020, we have rented a hosted Matrix setup at Swiss hosting company
+[unglei.ch](https://ungleich.ch/u/products/hosted-matrix-chat/). Our custom
+entry point [matrix.gnuradio.org](https://matrix.gnuradio.org) points to their
+server, but unlike Slack, we can pack up our chat and move it elsewhere if we're
+unhappy with the hosting service provided. The hosting service was chosen based
+on an internet research and a comparison of services provided, and cost. We have
+no other affiliations with the hosting company.
+
+We only recently set this up, and we probably want to tune this quite a bit. For
+example, we don't have any integrations set up such as the GitHub ticker, and
+we're not even sure if we need it. Join us on Matrix and give us your feedback!
```

Comment:
Explicitily address the IRC gateway; "we're still experimenting with the IRC gateway, which might be most confusing right now, as we haven't fully set it up like we want it to be, and we're also bridging over into slack."

Reply:
Also using this.